index.html

```diff
@@ -322,10 +322,13 @@ <h2 class="primary-text-about-me">About
         <h2 class="primary-text-contact-me"> Contact me</h2>
         <p class="secondary-text-contact">If you have an application you are interested in developing, a feature that
           you need built or a project that needs coding. I’d love to help with</p>
-        <input class="input" name="name" id="name" type="name" placeholder="Enter your name" pattern="[A-Za-z/ ]{0,30}"
+          <label for="name"></label>
+        <input class="input" name="user_name" id="name" type="name" placeholder="Enter your name" pattern="[A-Za-z/ ]{0,30}"
           required>
-        <input class="input" name="Email" id="email" type="email" placeholder="Enter your email" required>
-        <textarea class="input" name="message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
+          <label for="email" id="email"></label>
+        <input class="input" name="user_email" id="email" type="email" placeholder="Enter your email" required>
+        <label for="message"></label>
+        <textarea class="input" name="user_message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
         <button class="submit-button" id="form-submit" type="submit">Get in touch</button>
       </form>
```

Comment:
- [ ]  I see 😃that there is a problem with the contrast color here at the li, so kindly try to fix this contrast of all li of languages where The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.😊

| Look Here|
| --- |
|   ![image](https://user-images.githubusercontent.com/83360636/216159834-0daedf82-328a-4f76-8803-a3c94afc428d.png) |

Reply:
## Hi @osamaashraf6 🖐

I kindly ask you to help me with your opinion. 😊

I was playing with the contrasts and I found the solution that passes several tests but the color varies a little with the figma design.

> My question is:

Is there any problem if I use this contrast?

![imagen](https://user-images.githubusercontent.com/90431831/216169880-8e0b23a4-fad6-46ec-92b8-80c2b70e826f.png)

Reply:
> Correction

|Get in touch|Test contrast color|
|:---------:|:-------------------:|
|![imagen](https://user-images.githubusercontent.com/90431831/216196057-034d2618-7512-45ff-b220-c2c9bb4686d4.png)|![imagen](https://user-images.githubusercontent.com/90431831/216195953-f5706ea0-b2ca-460f-9c52-65cb5fdf79c2.png)|